### PR TITLE
Add time blog post

### PR DESCRIPTION
**Pull request by vn-ki**
As I promised. 

Sorry for the delay.

:cake:

### CODE REVIEW

File: posts/2018/2018-10-20-time.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
+Transition from `datetime` to astropy Time
```
Comment (nabobalis) [inline posts/2018/2018-10-20-time.rst]: `astropy.time` so it links

File: posts/2018/2018-10-20-time.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+==========================================
+
+.. post:: Oct 20, 2018
+   :author: Vishnunarayan K I
+   :tags: sunpy, time
+   :category: Release
```
Comment (nabobalis) [inline posts/2018/2018-10-20-time.rst]: GSoC I think this should go under.

File: posts/2018/2018-10-20-time.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+The SunPy project is very happy to announce that the time handling in SunPy has
```
Comment (nabobalis) [inline posts/2018/2018-10-20-time.rst]: has -> will

File: posts/2018/2018-10-20-time.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
+Transition from `datetime` to astropy Time
+==========================================
+
+.. post:: Oct 20, 2018
+   :author: Vishnunarayan K I
+   :tags: sunpy, time
+   :category: Release
+
+The SunPy project is very happy to announce that the time handling in SunPy has 
+been transitioned from python `datetime` module to `astropy.time` module.
```
Comment (nabobalis) [inline posts/2018/2018-10-20-time.rst]: remove `been`.

File: posts/2018/2018-10-20-time.rst [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
```
Comment (nabobalis) [inline posts/2018/2018-10-20-time.rst]: Add a new line saying that these changes are in a PR with a link. And you the reader can review the changes there

File: posts/2018/2018-10-20-time.rst [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+This comes with numerous benefits like:
```
Comment (nabobalis) [inline posts/2018/2018-10-20-time.rst]: remove `like`

File: posts/2018/2018-10-20-time.rst (hunk 5 above)
Comment (nabobalis) [reply posts/2018/2018-10-20-time.rst]: Also that these changes will show up in 1.0

File: posts/2018/2018-10-20-time.rst [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+- **Changes to** `~sunpy.time.parse_time`
+
+  `~sunpy.time.parse_time` has been reduced to a tiny wrapper over `~astropy.time.Time`. 
+  The API of `~sunpy.time.parse_time` is mostly similar to that of `~astropy.time.Time`. 
+  `~sunpy.time.parse_time` supports parsing a few more formats than `~astropy.time.Time`, which
+  are `numpy.datetime64`, `pandas.Series`, `pandas.DatetimeIndex`, utime and a few other time formats.
+
+
+| The work on the transtion from `datetime` to `astropy.time.Time` was done as a part of 
+| Vishnunarayan's Google Summer of Code 2018 project.
```
Comment (nabobalis) [inline posts/2018/2018-10-20-time.rst]: Could link to your medium posts?

**Comment by nabobalis** [comment]
Thanks @vn-ki! Just a few minor changes. 

@Cadair will want to have a read over this sometime!

**Comment by vn-ki** [comment]
@nabobalis Done!

**Comment by nabobalis** [comment]
Looks good. Here is the link to the formated post: https://deploy-preview-138--sunpy.netlify.com/posts/2018/2018-10-20-time.html

**Pull request merged**